styles/style.css

```diff
@@ -0,0 +1,2134 @@
+  right: 1rem;
+  padding: 0.5rem 1rem;
+  color: var(--primary-text-color);
+  font-weight: 500;
+  text-transform: uppercase;
+  background-color: var(--primary-color);
+  border-radius: 6.25rem;
+}
+
+.label__new {
+  background-color: #007772;
+}
+
+.label__sold-out {
+  background-color: #fff;
+  color: var(--fade-color);
+}
+
+.input {
+  border-radius: 0.5rem;
+  border: 0.075rem solid var(--secondary-color);
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  min-width: calc(15rem - 2 * 0.75rem - 0.1rem);
+  transition: var(--standart-transition);
+}
+
+.input::placeholder {
+  color: var(--fade-color);
+}
+
+.input:hover {
+  border: 0.075rem solid var(--color-focus);
+  box-shadow: 0.075rem 0.075rem var(--color-focus),
+    -0.075rem 0.075rem var(--color-focus), 0.075rem -0.075rem var(--color-focus),
+    -0.075rem -0.075rem var(--color-focus);
+}
+
+.input:focus {
+  border: 0.075rem solid var(--color-focus);
+  box-shadow: var(--focus-shadow);
+}
+
+input:invalid:not(:focus) {
+  border: 0.1rem solid var(--invalid-color);
+}
+
+.contact-form__field:has(.input:invalid)::after {
+  content: "This field is required.";
+  position: absolute;
+  backface-visibility: hidden;
+  top: 100%;
+  left: 0;
+  color: var(--invalid-color);
+  font-size: 0.75rem;
+  opacity: 0;
+  transition: border 0.3s ease-out, box-shadow 0.3s ease-out,
+    opacity 0.3s ease-out;
+  text-transform: none;
+}
+
+.contact-form__field:has(.input:invalid:not(:focus))::after {
+  opacity: 1;
+}
+
+.contact-form__field:has(.input[required]:valid)::after {
+  content: "Field validation is successful.";
+  position: absolute;
+  backface-visibility: hidden;
+  top: 100%;
+  left: 0;
+  color: #008f5d;
+  font-size: 0.75rem;
+  opacity: 0;
+  transition: border 0.3s ease-out, box-shadow 0.3s ease-out,
+    opacity 0.3s ease-out;
+  text-transform: none;
+}
+
+.contact-form__field:has(.input[required]:valid:focus)::after {
+  opacity: 1;
+}
+
+.input__large {
+  padding: 1.125rem 1.375rem;
+}
+
+.input__small {
+  padding: 0.625rem 0.75rem;
+}
+
+.input__form {
+  padding: 1.2rem 1.125rem;
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.625rem 5rem;
+  gap: 2rem;
+}
+
+.header__title,
+.header__logo {
+  font-size: 0;
+  line-height: 0;
+}
+
+.header__navigation {
+  flex-grow: 2;
+  height: fit-content;
+}
+
+.header__assets-block {
+  display: flex;
+}
+
+.main {
+  display: flex;
+  grid-template-columns: 9.5rem 1fr;
+  max-width: 100%;
+  justify-content: space-between;
+  padding: 7.5rem 52.5rem 7.5rem 5rem;
+  position: relative;
+}
+
+.main::after {
+  content: "";
+  position: absolute;
+  width: var(--screen-width);
+  left: calc((100% - (var(--screen-width))) / 2);
+  top: 0;
+  height: 100%;
+  z-index: -2;
+  background: linear-gradient(to right, var(--background-color) 52%, #fff 86%);
+}
+
+.main::before {
+  content: "";
+  position: absolute;
```

Comment:
3.12. ❤️ Use flex or grid for layouts.
Do not use tables or positioning.

The container(.main) is already a Flex container, use it please.

Reply:
Not fixed

Reply:
Fixed 🥇